### DISCUSSION TITLE
Feeds are not loading after update to 1.22.1

### DISCUSSION THREAD

**Discussion by OlliM** [FreshRSS/FreshRSS#5800] [Q&A] [answered]
**Describe the bug**
All my feeds are broken, showing 
`Blast! This feed has encountered a problem. Please verify that it is always reachable then update it.`

**Expected behavior**
Feeds should load.

**Environment information (please complete the following information):**
- Docker with latest image (1.22.1)

**Additional context**

Here's the error from the logs:
`cURL error 6: getaddrinfo() thread failed to start [https://www.engadget.com/rss.xml]`

**Comment by OlliM** [comment, marked as answer]
Updating Docker from 20.x to 24.x resolved the issue

**Reply by Alkarex** [reply]
See same question https://github.com/FreshRSS/FreshRSS/discussions/5742#discussioncomment-7382612
What is your operative system?
Another option is to try `freshrss/freshrss:alpine`
